### DISCUSSION TITLE
How do we document the fields and their data type for JSON payloads in REST APIs ?

### DISCUSSION THREAD

**Discussion by tmcdos** [firecamp-dev/firecamp#198] [General]
How do we document the JSON schema for the JSON payloads in REST APIs ?

**Comment by Nishchit14** [comment]
It's not available atm, but that would be a great feature, @tmcdos would you be happy to raise a feature request for the same. I would be happy to discuss more about it.

**Reply by tmcdos** [reply]
https://github.com/firecamp-dev/firecamp/discussions/200
